src/pybind/mgr/HACKING.md

```diff
@@ -0,0 +1,56 @@
+# Ceph Manager Developer Documentation
+
+## Unit Testing
+
+In the manager our unit tests are based on **tox**.
+
+> **Note:**
+> Currently only [doctests](https://docs.python.org/3/library/doctest.html) inside
+> `mgr_util.py` are run
+
+### How to run the tests
```

Comment:
I don't think we need this section anymore at this place. With the project global `src/script/run_tox.sh`, this could potentially be moved to `/doc/`. I for one would prefer to have no documentation here, instad of some outdated one.